branchpro/apps/_simulation.py

```diff
@@ -49,33 +63,69 @@ def __init__(self):
-                    dcc.Upload(
-                        id='upload-data',
-                        children=html.Div([
-                            'Drag and Drop or ',
-                            html.A('Select Files',
-                                   style={'text-decoration': 'underline'}),
-                            ' to upload your Incidence Number data.'
-                        ]),
-                        style={
-                            'width': '100%',
-                            'height': '60px',
-                            'lineHeight': '60px',
-                            'borderWidth': '1px',
-                            'borderStyle': 'dashed',
-                            'borderRadius': '5px',
-                            'textAlign': 'center',
-                            'margin': '10px'
-                        },
-                        multiple=True  # Allow multiple files to be uploaded
+                    dbc.Row(
+                        [
+                            dbc.Col(
+                                children=[
+                                    html.H4([
+                                        'You can upload your own '
+                                        'incidence data here.'
+                                    ]),
+                                    html.Div([
+                                        'It will appear as bars, while'
+                                        ' the simulation will be a line.'
+                                    ]),
+                                    dcc.Upload(
+                                        id='upload-data',
+                                        children=html.Div([
+                                            'Drag and Drop or ',
+                                            html.A(
+                                                'Select Files',
+                                                style={
+                                                    'text-decoration':
+                                                    'underline'}),
+                                            ' to upload your Incidence Number '
+                                            'data.'
+                                        ]),
+                                        style=button_style,
+                                        # Allow multiple files to be uploaded
+                                        multiple=True
+                                    ),
+                                    html.Div(id='incidence-data-upload')]),
+                            dbc.Col(
+                                children=[
+                                    html.H6([
+                                        'You can upload your own serial \
+                                            interval here.'
+                                    ]),
+                                    html.Div([
+                                        'Data must contain one serial '
```

Comment:
This upload does not prohibit files with multiple serial intervals. Let's just add a brief note here saying that if multiple serial intervals are uploaded, the first one will be used.